1 intro tast/MathController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  MathController.swift
+//  1 intro tast
+//
+//  Created by Belka on 2/2/20.
+//  Copyright © 2020 Tanya. All rights reserved.
+//
+
+class MathController {
+    //The mean (average) is found by adding all of the numbers together and dividing by the number of items in the set. Example: 10 + 10 + 20 + 40 + 70 / 5 = 30.
```

Comment:
nit: try to put the space between `//` and the comment:
```suggestion
    // The mean (average) is found by adding all of the numbers together and dividing by the number of items in the set. Example: 10 + 10 + 20 + 40 + 70 / 5 = 30.
```